spec/models/instance/settings_spec.rb

```diff
@@ -1,45 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Instance::Settings, type: :model do
-  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F]
-  ENABLED_COMPONENTS = [:A, :C, :E]
+  ALL_COMPONENTS = [:A, :B, :C, :D, :E, :F].freeze
```

Comment:
Oh I get you, so shall we always use `let` ? 

The are problems when we define another constant with the same name again.